dss/operations/lambda_params.py

```diff
@@ -0,0 +1,290 @@
+
+from dss.operations import dispatch
+from dss.operations.ssm_params import get_ssm_variable_prefix, fix_ssm_variable_prefix
+from dss.operations.ssm_params import get_ssm_parameter, set_ssm_parameter
+
+from dss.util.aws.clients import ssm as ssm_client  # type: ignore
+from dss.util.aws.clients import secretsmanager as sm_client  # type: ignore
+from dss.util.aws.clients import es as es_client  # type: ignore
+import dss.util.aws.clients
+lambda_client = getattr(dss.util.aws.clients, "lambda")
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_elasticsearch_endpoint() -> str:
+    domain_name = os.environ["DSS_ES_DOMAIN"]
+    domain_info = es_client.describe_elasticsearch_domain(DomainName=domain_name)
+    return domain_info["DomainStatus"]["Endpoint"]
+
+
+def get_admin_emails() -> str:
+
+    def get_secret_variable_prefix() -> str:
+        # TODO: this functionality should be moved to secrets.py
+        store_name = os.environ["DSS_SECRETS_STORE"]
+        stage_name = os.environ["DSS_DEPLOYMENT_STAGE"]
+        store_prefix = f"{store_name}/{stage_name}"
+        return store_prefix
+
+    def fix_secret_variable_prefix(secret_name: str) -> str:
+        # TODO: this functionality should be moved to secrets.py
+        prefix = get_secret_variable_prefix()
+        if not (secret_name.startswith(prefix) or secret_name.startswith("/" + prefix)):
+            secret_name = f"{prefix}/{secret_name}"
+        return secret_name
+    
+    def fetch_secret_safely(secret_name: str) -> dict:
+        # TODO: this functionality should be moved to secrets.py
+        try:
+            response = sm_client.get_secret_value(SecretId=secret_name)
+        except ClientError as e:
+            if 'Error' in e.response:
+                errtype = e.response['Error']['Code']
+                if errtype == 'ResourceNotFoundException':
+                    raise RuntimeError(f"Error: secret {secret_name} was not found!")
+            raise RuntimeError(f"Error: could not fetch secret {secret_name} from secrets manager")
+        else:
+            return response
+
+    gcp_var = "GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME"
+    gcp_secret_id = fix_secret_variable_prefix(gcp_var)
+    response = fetch_secret_safely(gcp_secret_id)['SecretString']
+    gcp_service_account_email = json.loads(response)['client_email']
+
+    admin_var = "ADMIN_USER_EMAILS_SECRETS_NAME"
+    admin_secret_id = fix_secret_variable_prefix(admin_var)
+    response = fetch_secret_safely(admin_secret_id)['SecretString']
+    email_list = [email.strip() for email in response.split(',') if email.strip()]
+    
+    if gcp_service_account_email not in email_list:
+        email_list.append(gcp_service_account_email)
+    return ",".join(email_list)
+
+
+def get_deployed_lambdas(quiet: bool = True):
+    """
+    Generator returning names of lambda functions
+
+    :param quiet: (boolean) if true, don't print warnings about lambdas that can't be found
```

Comment:
Nit: this could be written instead as
```suggestion
    :param bool quiet: if true, don't print warnings about lambdas that can't be found
```